cypress/e2e/7-workflow-actions.cy.ts

```diff
@@ -36,7 +37,7 @@ describe('Workflow Actions', () => {
 		WorkflowPage.getters.isWorkflowSaved();
 	});
 
-	it('should not save already saved workflow', () => {
+	it.skip('should not save already saved workflow', () => {
```

Comment:
test failing on master. @MiloradFilipovic is looking into this. https://github.com/n8n-io/n8n/actions/runs/9453100846/job/26037911945

Reply:
Please ensure that this doesn't get left in the end